### PR TITLE
Update getting-started.adoc

### PR DESCRIPTION
**Pull request by LichKing-lee**
Add webtestclient in gradle

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@LichKing-lee Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-restdocs&pullRequestId=654)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-restdocs&pullRequestId=654) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@LichKing-lee Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-restdocs&pullRequestId=654)!

File: docs/src/docs/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -172,9 +172,9 @@ the configuration are described in the following listings:
 	This will automatically configure the `snippets` attribute for use in your `.adoc`
 	files to point to `build/generated-snippets`. It will also allow you to use the
 	`operation` block macro.
-<3> Add a dependency on `spring-restdocs-mockmvc` in the `testCompile` configuration. If
-	you want to use REST Assured rather than MockMvc, add a dependency on
-	`spring-restdocs-restassured` instead.
+<3> Add a dependency on `spring-restdocs-mockmvc` in the `test` scope. If you want to use
```
[the remark this answers is not in the document]
Comment (LichKing-lee) [reply docs/src/docs/asciidoc/getting-started.adoc]: thanks. fix with maven

**Comment by wilkinsona** [comment]
@LichKing-lee Thank you very much for making your first contribution to Spring REST Docs. The proposed changes have been merged into master.

**Pull request closed without merging**